homeassistant/components/habitica/strings.json

```
@@ -25,7 +25,13 @@
     "tag_options_name": "Tags",
     "tag_options_description": "Add or remove tags from a task.",
     "name_description": "The title for the Habitica task.",
-    "cost_name": "Cost"
+    "cost_name": "Cost",
+    "difficulty_name": "Difficulty",
+    "difficulty_description": "Sets the task difficulty.",
+    "frequency_name": "Counter reset",
+    "frequency_description": "Defines when the habit's counter resets: daily at the start of a new day, weekly after Sunday night, or monthly at the beginning of a new month.",
```

```suggestion
    "frequency_description": "The frequency at which the habit's counter resets: daily at the start of a new day, weekly after Sunday night, or monthly at the beginning of a new month.",
```
also to try to match the other descriptions wording.